alfalfa_worker/worker_fmu/step_fmu.py

```diff
@@ -155,6 +154,8 @@ def run(self):
                 message = self.redis_pubsub.get_message()
                 if message:
                     data = message['data']
+                    if isinstance(data, bytes):
```

Comment:
can you speak to why this needs to happen? Is it comment worthy?

Reply:
For some reason I was getting `b'advance` as the data. I wonder if it has something to do with switching from python 2 to 3 that that now strings are coming in as `bytes`